.eslintrc.typed.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
```

Comment:
What's the point of having two lint-config-files? Seems to me that there are typescript rules in both?

Reply:
It's what I mentioned in the OP about linting with type-checking. `.eslintrc.js` is the normal linting, which is faster. `.eslintrc.typed.js` is the linting with type-checking. It compiles the entire project with TypeScript and uses the type information to apply the rules. The different is pretty much here: https://github.com/trakt-tools/universal-trakt-sync/blob/prettier-eslint/.eslintrc.typed.js#L33 This allows for a more robust linting, but it's slower, hence why it's only applied in the pre-commit hook.